tests/test_non_host.py

```diff
@@ -39,11 +39,3 @@ def test_custom_interpreter(
     if sys.version_info >= (3, 12):
         expected -= {"setuptools", "wheel"}
     assert found == expected, out
-
-    monkeypatch.setattr(sys, "argv", [*cmd, "--graph-output", "something"])
```

Comment:
We now can use --graph-output too, so I think we don't need this check anymore.

Reply:
Yes, since that test snippet covers the now outdated graphviz check, and all we're doing now is running the host python but with the custom python paths it makes sense to remove it.